app/busy.go

```diff
@@ -109,11 +110,12 @@ func (b *Busy) notifyServerBusyChange(sbs *model.ServerBusyState) {
 	if b.cluster == nil {
 		return
 	}
+	buf, _ := json.Marshal(sbs)
```

Comment:
I didn't want to check the error just yet. Kept the scope of the PR just to changing the field type.

Reply:
Let's just make sure to track this.

Reply:
https://mattermost.atlassian.net/browse/MM-37409